app/components/publications/overview/publications-table-row.js

```diff
@@ -53,13 +61,44 @@ export default class PublicationsTableRowComponent extends Component {
     return this.publicationStatus && getPublicationStatusPillStep(this.publicationStatus);
   }
 
+  /** @returns {Date?} undefined if no translation-activities */
+  async getTranslationRequestDate(publicationFlow) {
+    const publicationSubcase = await publicationFlow.translationSubcase;
+    const translationActivities = await publicationSubcase.translationActivities;
+    const requestDates = translationActivities.mapBy('startDate');
+    const mostRecentDate = this.getMaxDate(requestDates);
+    return mostRecentDate;
+  }
+
+  /** @returns {Date?} undefined if no ProofingActivities */
   async getProofRequestDate(publicationFlow) {
     const publicationSubcase = await publicationFlow.publicationSubcase;
     const proofingActivities = await publicationSubcase.proofingActivities;
-    const startDates = proofingActivities.mapBy('startDate');
-    startDates.sort();
-    const firstProofRequestDate = startDates[0];
-    return firstProofRequestDate;
+    const requestDates = proofingActivities.mapBy('startDate');
+    const mostRecentDate = this.getMaxDate(requestDates);
+    return mostRecentDate;
+  }
+
+  getMaxDate(dates) {
+    // default .sort() of Date objects does not give expected results
```

Comment:
Wat bedoel je hiermee?  Wat je in de andere comment zegt dat `undefined` achteraan gesorteerd wordt? Is het dan niet duidelijker om die lege values eruit te filteren en dan de laatste waarde te nemen?

Reply:
Het is een ander issue: ik verwachtte dat `dates.sort()` zou sorteren van oudst naar recentst, maar dat is niet zo.
[example JSFiddle](https://jsfiddle.net/x9tokabg/3/)